pages/docs/guides/validate.md

```diff
@@ -0,0 +1,99 @@
+---
```

Comment:
As a side note, since you are repurposing this as a `guide`, it would have been ideal if you had remembered to review this document for the word **tutorial** so you could update it correctly. 😜 

@Annysah

Reply:
> As a side note, since you are repurposing this as a `guide`, it would have been ideal if you had remembered to review this document for the word **tutorial** so you could update it correctly. stuck_out_tongue_winking_eye
> 
> @Annysah

Oops !😬. I actually did but completely didn't look at what I had in my meta-data 🤦‍♀️

Reply:
> Heyooo @Annysah! here's what I'm going to do here... I will rewrite this a little to show you how information architecture works when writing `guides` content and also to give an editorial review. This now NEW `guide` will have an outline different than what we have in tutorials because **the purpose of a `guide` is to explain a tech's capabilities and features at a high level**.
> 
> For `guides`, let's shoot for the following outline structure:
> 
> * Introduction
> * Prerequisites (dev setup/tools users need in described scenarios covered)
> * SCENARIO A
> * SCENARIO B
> * SCENARIO C
> * Additional Resources _(when applicable)_

Will the scenarios in this case be the "Validation against the specification" and "Validation against the company's best practices..."?

Reply:
exactly !